dbt/loader.py

```diff
@@ -121,6 +126,35 @@ def load_project(cls, root_project, all_projects, project, project_name,
 
 
 class SchemaTestLoader(ResourceLoader):
+    @classmethod
+    def load_all(cls, root_project, all_projects, macros=None):
+        tests = {}
+        patches = {}
+        for project_name, project in all_projects.items():
+            project_tests, project_patches = cls.load_project(
+                root_project, all_projects, project, project_name, macros
+            )
+            for unique_id, test in project_tests.items():
+                if unique_id in tests:
+                    dbt.exceptions.raise_duplicate_resource_name(
+                        test, tests[unique_id],
+                    )
+                tests[unique_id] = test
+
+            for name, patch in project_patches.items():
+                if name in patches:
+                    dbt.exceptions.raise_compiler_error(
+                        'dbt found two patches for the same model named {}. '
```

Comment:
I think we should use terminology besides "patch" in log/warning/error message. I don't know that it's going to be super clear to users what a patch is, but I think the second part of this message is great!

Reply:
Descriptions instead of patches?